[2021-07-09 04:36:04,910] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.PostgresToDataLake.extract_user_log_data 2021-07-09T04:36:02.713158+00:00 [queued]>
[2021-07-09 04:36:04,926] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.PostgresToDataLake.extract_user_log_data 2021-07-09T04:36:02.713158+00:00 [queued]>
[2021-07-09 04:36:04,927] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-09 04:36:04,928] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-09 04:36:04,928] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-09 04:36:04,946] {taskinstance.py:1107} INFO - Executing <Task(PostgresToGCSOperator): PostgresToDataLake.extract_user_log_data> on 2021-07-09T04:36:02.713158+00:00
[2021-07-09 04:36:04,951] {standard_task_runner.py:52} INFO - Started process 9130 to run task
[2021-07-09 04:36:04,957] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'bluepi_dag', 'PostgresToDataLake.extract_user_log_data', '2021-07-09T04:36:02.713158+00:00', '--job-id', '333', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bluepi_dag.py', '--cfg-path', '/tmp/tmpro67s3mz', '--error-file', '/tmp/tmp7pxhvq5b']
[2021-07-09 04:36:04,965] {standard_task_runner.py:77} INFO - Job 333: Subtask PostgresToDataLake.extract_user_log_data
[2021-07-09 04:36:05,023] {logging_mixin.py:104} INFO - Running <TaskInstance: bluepi_dag.PostgresToDataLake.extract_user_log_data 2021-07-09T04:36:02.713158+00:00 [running]> on host 3cf0296804bd
[2021-07-09 04:36:05,084] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=natchanon
AIRFLOW_CTX_DAG_ID=bluepi_dag
AIRFLOW_CTX_TASK_ID=PostgresToDataLake.extract_user_log_data
AIRFLOW_CTX_EXECUTION_DATE=2021-07-09T04:36:02.713158+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-09T04:36:02.713158+00:00
[2021-07-09 04:36:05,087] {sql_to_gcs.py:150} INFO - Executing query
[2021-07-09 04:36:05,097] {base.py:78} INFO - Using connection to: id: my_postgres_conn. Host: 35.247.174.171, Port: 5432, Schema: postgres, Login: exam, Password: ***, extra: {}
[2021-07-09 04:36:06,637] {sql_to_gcs.py:153} INFO - Writing local data files
[2021-07-09 04:36:06,638] {sql_to_gcs.py:202} INFO - Current file count: 1
[2021-07-09 04:36:06,639] {sql_to_gcs.py:164} INFO - Uploading 1 files to GCS.
[2021-07-09 04:36:06,910] {gcs.py:485} INFO - File /tmp/tmpf4866841 uploaded to raw_data/user_log in bluepi-landing-bucket bucket
[2021-07-09 04:36:06,911] {sql_to_gcs.py:167} INFO - Removing local files
[2021-07-09 04:36:06,924] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=bluepi_dag, task_id=PostgresToDataLake.extract_user_log_data, execution_date=20210709T043602, start_date=20210709T043604, end_date=20210709T043606
[2021-07-09 04:36:06,957] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-09 04:36:06,976] {local_task_job.py:151} INFO - Task exited with return code 0
